### ISSUE TITLE
Monitoring error. Any stderr from `du -mL` command should be filtered.

### ISSUE THREAD

**Issue by kvaps** [OpenNebula/one#1525]
# Bug Report
## Version of OpenNebula
- [ ] 5.2.2
- [ ] 5.4.0
- [ ] 5.4.1
- [ ] 5.4.2
- [X] 5.4.3
- [ ] Development build

## Component
- [ ] Authorization (LDAP, x509 certs...)
- [ ] Command Line Interface (CLI)
- [ ] Contextualization
- [ ] Documentation
- [ ] KVM
- [ ] Networking
- [ ] Orchestration (OpenNebula Flow)
- [ ] Sunstone
- [X] Storage
- [ ] Scheduler
- [ ] vCenter

### Expected Behavior
When physical disks removed, symlinks on the system storage can not be removed for some reason.
If such symlink is exist somewhere, this breaks the whole monitoring script output.
Monitoring script should return clean information for each host.
And even `du` have some stderr stream it should be filtered.

### Actual Behavior
All stderr returned with monitoring information so opennebula can not parse this outuput and we have errors like these:

```
Sat Nov 25 20:45:57 2017 [Z0][ONE][E]: Error parsing host information: syntax error, unexpected VARIABLE, expecting EQUAL or EQUAL_EMPTY at line 1, columns 5:11. Monitoring information:
du: cannot access '/var/lib/one//datastores/101/5511/disk.3'
du: cannot access '/var/lib/one//datastores/101/5527/disk.3'
du: cannot access '/var/lib/one//datastores/101/5528/disk.4'
du: cannot access '/var/lib/one//datastores/101/5534/disk.6'
du: cannot access '/var/lib/one//datastores/101/5535/disk.3'
ARCH=x86_64
MODELNAME="Intel(R) Xeon(R) CPU E3-1284L v4 @ 2.90GHz"
HYPERVISOR=kvm
TOTALCPU=800
CPUSPEED=3700
TOTALMEMORY=32872952
USEDMEMORY=10102652
FREEMEMORY=22770300
...
```

## How to reproduce

just run this command:
```
ln -s /tmp/blabla /var/lib/one/datastores/101/5534/disk.9
```
where:
* `101` - it is system datastore
* `5534` - some vm
* `disk.9` - non existing disk
* `/tmp/blabla` - non existing file

**Comment by kvaps** [comment]
PR here:
https://github.com/OpenNebula/one/pull/1527

**Comment by rsmontero** [comment]
This needs to be backported to 5.4 branch

**Comment by vholer** [comment]
Thank you for the PR, I took your patch and added handling of the stderr for few more commands. The fix will be part of #1361. Closing this one.
